tests/pkg_nanopb/README.md

```diff
@@ -0,0 +1,29 @@
+# Introduction
+
+This is a test application for the nanoPb library.
+The library provides a Google Protocol Buffers encoder / decoder.
+
+# Prerequisites
+
+Install the protobuf compiler and the protobuf python bindings.
+On Debian/ubuntu, the corresponding packages are `protobuf-compiler` and
+`python-protobuf`. On Arch, it is `protobuf` and `python-protobuf`.
+
+You'll also need the nanopb protoc generator plugin.
+
+Checkout the matching nanopb repository:
+
+    $ git clone https://github.com/nanopb/nanopb nanopb-0.3.9.3
```

Comment:
Why do we have to check out the nanopb repository twice?
It's already checked out by the build system to `bin/pkg/$(BOARD)/nanopb/generator` - can't we just use that?